test/new-e2e/tests/orchestrator/README.md

```diff
@@ -0,0 +1,113 @@
+# Prerequisites
```

Comment:
<!-- Praises highlight something positive. Try to leave at least one of these comments per review. Do look for something to sincerely praise. -->
**👏 praise**
Nice !